package.json

```
@@ -9,6 +9,7 @@
     "test:watch": "npm test -- --watch",
     "lint:js": "eslint .",
     "lint:css": "stylelint 'blocks/**/*.css' 'styles/*.css'",
+    "lint:less": "stylelint 'blocks/**/*.less' 'styles/*.less'",
     "lint": "npm run lint:js",
```

Probably we should enable linting checks once the lint errors have been fixed across `less` files.
```suggestion
    "lint": "npm run lint:js & npm run lint:less",
```

---

Yes good idea and once we have addressed the less linting issues we can also enable the css linting on the default `lint` runs.